src/components/NewGalaxyModal.js

```diff
@@ -306,17 +316,72 @@ export const NewGalaxyModal = _.flow(
                 onlyInteger: true,
                 value: dataDiskSize,
                 style: { marginTop: '0.5rem', width: '5rem' },
+                disabled: !!currentDataDisk,
+                tooltip: !!currentDataDisk ?
+                  ['Persistent disk of size ', currentDataDisk.size, ' GB already exists and will be attached upon Galaxy creation'] :
+                  undefined,
                 onChange: v => setDataDiskSize(v)
               })
             ])
           ])
         ])
       ])
+  }
+
+  const renderDeleteDiskChoices = () => {
+    return div({ style: { ...styles.drawerContent, ...styles.warningView } }, [
+      h(TitleBar, {
+        style: styles.titleBar,
+        title: h(WarningTitle, ['Delete environment options']),
+        onDismiss,
+        onPrevious: () => {
+          setViewMode(undefined)
+          setDeleteDiskSelected(false)
+        }
+      }),
+      div({ style: { lineHeight: '1.5rem' } }, [
+        h(Fragment, [
+          h(RadioBlock, {
+            name: 'delete-persistent-disk',
+            labelText: 'Keep persistent disk, delete application configuration and compute profile',
+            checked: !deleteDiskSelected,
+            onChange: () => setDeleteDiskSelected(false),
+            style: { marginTop: '1rem' }
+          }, [
+            p([
+              'Deletes your application configuration and cloud compute profile, but detaches your persistent disk and saves it for later. ',
+              'The disk will be automatically reattached the next time you create a Galaxy application.'
+            ]),
+            p({ style: { marginBottom: 0 } }, [
+              'You will continue to incur persistent disk cost at ',
+              span({ style: { fontWeight: 600 } }, [Utils.formatUSD(getGalaxyDiskCost(dataDiskSize)), ' per month.'])
```

Comment:
```suggestion
              span({ style: { fontWeight: 600 } }, [Utils.formatUSD(getGalaxyDiskCost(dataDiskSize)), ' per hour.'])
```
since `getGalaxyDiskCost()` returns hourly disk cost.